changelogs/unreleased/net.box-request-result-contains-a-copy-of-the-internal-error.md

```diff
@@ -0,0 +1,3 @@
+## feature/net.box
+
+* Result of request contains a copy of the internal error.
```

Comment:
Before commit 954194a1ca5c2cd1826a8a689663c827313dbbef, the error was copied in Lua, and we never officially released a build with that commit (only beta) so this changelog isn't needed. Please drop.

Reply:
Done.